src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+/** A recipe and its attributes. */
+public final class Recipe {
+
+  private final int id;
+  private final String title;
+  private final String link;
+  private final int likes;
+  private final String category;
+
+  public Task(int id, String title, String link, int likes, String category) {
```

Comment:
This should be Recipe

Reply:
Fixed. The other error and the nit I believe were previously fixed in the initial batch of commits for the pull request.

Reply:
I think you might be right about the changes previously made, sorry about that! Looks like I'm still a GitHub UI novice...haha